executor/ddl.go

```diff
@@ -821,7 +821,7 @@ func (e *DDLExec) getRecoverTableByTableName(tableName *ast.TableName) (*model.J
 		return nil, nil, err
 	}
 	if tableInfo == nil || jobInfo == nil {
-		return nil, nil, errors.Errorf("Can't find dropped/truncated table: %v in DDL history jobs", tableName.Name)
+		return nil, nil, errors.Errorf("Can't find localTemporary/dropped/truncated table: %v in DDL history jobs", tableName.Name)
```

Comment:
I don't think this modification is necessary. It may mislead users that local temporary table should be found in ddl history jobs but not.

Reply:
The point is we can not distinguish a dropped local temporary table from a non-existent table if we do not cache the dropped localTemporary. 
Only in DDL history jobs that we can flashback/recover a table, local temporary is not in the jobs, so it can not be flashback/recover. I think the user doesn't care about if it shoud be.

Reply:
Emm.. My opinion that we should clarify our behavior that local temporary table will not be recovered if it is dropped. These error message may make user think that "local temporary should be found in ddl history jobs but not" and may regard it as bug. Just a personal opinion...

Reply:
I think it's fine to remove this part and just report `errUnsupportedFlashbackTmpTable` just like global temporary tables. The behavior is consistent and easy to understand: we just don't support that.